src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -161,7 +161,9 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
 
             if (options.skipSDKGeneration !== true) {
                 const csProjFile = resolvePath(
-                    outputFolder,
+                    outputFolder.endsWith("src")
```

Comment:
is there a way with more semantic to assert the path has a directory or at least a segment of `src`?
Because our current implementation catches paths like `mysrc` which I think is not our purpose.

Reply:
Yes, we need to handle this. I will resolve in another PR which will contain both `emitter` and `autorest.csharp codegen` sides.